src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -685,7 +685,33 @@ private Result ConstructPathTraversalResult(ContrastLogReader.Context context)
 
             // default : Attacker-controlled path traversal was observed from '{0}' on '{1}' page.
 
-            return ConstructNotImplementedRuleResult(context.RuleId);
+            Result result = CreateResultCore(context);
+
+            result.Locations = new List<Location>
+            {
+                new Location
+                {
+                    LogicalLocation = new LogicalLocation
+                    {
+                        FullyQualifiedName = GetUserCodeLocation(context.MethodEvent.Stack)
+                    }
+                }
+            };
+
+            string page = context.RequestTarget;
+            string sources = BuildSourcesString(context.Sources);
+
+            result.Message = new Message
+            {
+                Id = "default",
+                Arguments = new List<string>
+                {              // Attacker-controlled path traversal was observed from
```

Comment:
>Attacker [](start = 34, length = 8)

Suggest adding an article, 'An attacker-controlled path...'

Reply:
Fixed. Glad I have leave to copy-edit the messages (which I assume came directly from CS). In that case, also "'{1}' page" => "page '{1}'".

---
In reply to: [282149268](https://github.com/microsoft/sarif-sdk/pull/1480#discussion_r282149268) [](ancestors = 282149268)